contracts/Collector.sol

```diff
@@ -9,18 +9,21 @@ import "@openzeppelin/contracts/math/SafeMath.sol";
 contract Collector is ICollector{
     IERC20 public token;
     address public owner;
+    address private remainderAddress;
     RevenuePartner[] private partners;
 
     constructor(
         address _owner,
         IERC20 _token,
-        RevenuePartner[] memory _partners
+        RevenuePartner[] memory _partners,
+        address _remainderAddress
```

Comment:
Do you see any addition in providing a method to update both partners and reminderAddress in one single transaction?

Reply:
Maybe we can do this in a separate task? I do think it makes sense to have this functionality available (through an overloaded function) but I faced an issue while testing it.